fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/InternalToCertificate.java

```diff
@@ -28,8 +34,26 @@
 public class InternalToCertificate {
 
     public static Certificate toCertificate(LuaenaUtlatandeV1 internalCertificate, CertificateTextProvider textProvider) {
+        int index = 0;
         return CertificateBuilder.create()
             .metadata(MetaDataGrundData.toCertificate(internalCertificate, textProvider))
+            .addElement(
```

Comment:
Just an observation. Earlier implementation with start with the arguments that are specific for the question itself and then follow with index and textProvider. Is it a specific reason or it just happened?